### PR TITLE
Update nidigital metadata

### LINKED ISSUE #940: Add NI Digital Pattern Driver support
* Description of issue

We need to add support for the NI Digital Pattern Driver

### PR DESCRIPTION
**Pull request by texasaggie97**
- [X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).
- [X] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.
- [X] ~~I've added tests applicable for this pull request~~

### What does this Pull Request accomplish?
* Updates to nidigital metadata
    * Multiple enums added
    * Private functions removed
* For every property, adds documentation that specifies the corresponding LabVIEW API property
### List issues fixed by this Pull Request below, if any.
* Fixes #940
### What testing has been done?
* flake8

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -55,8 +55,24 @@ All notable changes to this project will be documented in this file.
...
         * Looking for any testing and/or feedback
+        * `get_channel_name_from_string()`
     * #### Changed
+        *  New enums:
+
+           | Enum name                  | Where used                                                                       |
+           |----------------------------|----------------------------------------------------------------------------------|
+           | `DigitalEdge`              | `digital_edge_conditional_jump_trigger_edge`, `digital_edge_start_trigger_edge`  |
+           | `ApertureTimeUnits`        | `ppmu_aperture_time_units`, `ppmu_configure_aperture_time(units)`                |
+           | `PpmuOutputFunction`       | `ppmu_output_function`                                                           |
```
Comment (marcoskirsch) [inline CHANGELOG.md]: PPMU is an acronym, shouldn't we spell this PPMUOutputFunction? Or how do we treat other acronyms when they are in the beginning of a type?

File: CHANGELOG.md [hunk 2]
```diff
@@ -55,8 +55,24 @@ All notable changes to this project will be documented in this file.
...
+           | `SelectedFunction`         | `selected_function`                                                              |
+           | `TdrEndpointTermination`   | `tdr_endpoint_termination`                                                       |
```
Comment (marcoskirsch) [inline CHANGELOG.md]: TDR is an acronym, shouldn't we spell this TDREndpointTermination? Or how do we treat other acronyms when they are in the beginning of a type?

File: docs/nidigital/enums.rst [hunk 3]
```diff
@@ -6,6 +6,96 @@ Enums used in NI-Digital Pattern Driver
...
+ApertureTimeUnits
+-----------------
+
+.. py:class:: ApertureTimeUnits
+
+    .. py:attribute:: ApertureTimeUnits.SECONDS
+
+
+
+DigitalEdge
```
Comment (marcoskirsch) [inline docs/nidigital/enums.rst]: We should double check our conventions in the nimi-python APIs as far as enum naming goes. I think we use plural for the names.

If not explicitly stated in our wiki, we should.

File: CHANGELOG.md (hunk 1 above)
Comment (texasaggie97) [reply CHANGELOG.md]: From nidcpower `DCNoiseRejection` and from nidmm `ADCCalibration`

Change to PPMU*

File: CHANGELOG.md (hunk 2 above)
Comment (texasaggie97) [reply CHANGELOG.md]: Changed

File: docs/nidigital/enums.rst (hunk 3 above)
Comment (texasaggie97) [reply docs/nidigital/enums.rst]: From nidcpower `DigitalEdge`

I don't think we are consistent, nor do I think we can be. Some it makes sense to make plural and some it does not.

**Pull request merged**